js/contribution.js

```diff
@@ -21,8 +21,8 @@ const msecs = {
 class JS extends Contribution {
   constructor () {
     super()
-    this.paymentEnabled = true
-    this.paymentsAllowNonVerified = true
+    this.funding = 'sufficientFunds'
+    this.paymentsContributionAmount = 10
```

Comment:
Where is this one being used? Any reason we can't just keep it to `PAYMENTS_CONTRIBUTION_AMOUNT` ?

Reply:
It's unused. Removed.